ddpui/api/airbyte_api.py

```diff
@@ -619,3 +620,37 @@ def delete_airbyte_source_v1(request, source_id):
         raise HttpError(400, error)
 
     return {"success": 1}
+
+
+@airbyteapi.get(
+    "/v1/connections/{connection_id}/catalog", 
+    auth=auth.CustomAuthMiddleware(), 
+)
+@has_permission(["can_view_connection"])
+def get_connection_catalog_v1(request, connection_id):
+    """Fetch a connection in the user organization workspace"""
+    orguser: OrgUser = request.orguser
+    if orguser.org.airbyte_workspace_id is None:
+        raise HttpError(400, "create an airbyte workspace first")
+
+    res = airbytehelpers.get_connection_catalog(connection_id)
+    if res is None:
+        raise HttpError(400, "connection not found")
+    return res
+
+
+@airbyteapi.put(
+    "/v1/connections/{connection_id}/schema_update", auth=auth.CustomAuthMiddleware()
+)
+@has_permission(["can_edit_connection"])
+def update_schema_changes_connection(request, connection_id, payload: AirbyteConnectionSchemaUpdate):
```

Comment:
you can call this `update_connection_schema` or `apply_changes_to_connection_schema`